rts/lra-examples/lra-coordinator/pom.xml

```diff
@@ -87,5 +87,16 @@
             <artifactId>cdi</artifactId>
             <version>${version.wildfly-swarm}</version>
         </dependency>
+        <!-- generate REST API from swagger
```

Comment:
Is this necessary to leave in?

Reply:
As a quickstart I thought it would be useful for the reader to see what API our spec implementation uses to communicate with our coordinator. He may wish to experiment with non Java based clients and participants (nodejs for example).